lib/model/QubitNote.php

```diff
@@ -22,6 +22,27 @@ class QubitNote extends BaseNote
     // Flag for updating search index on save or delete
     public $indexOnSave = true;
 
+    public function __get($name)
+    {
+        $args = func_get_args();
+
+        $options = [];
+        if (1 < count($args)) {
+            $options = $args[1];
+        }
+
+        if ('type' === $name && true === $options['sourceCulture']) {
```

Comment:
Will this trigger an error if the 'sourceCulture' key is not present in the array?

Reply:
No, it just skips the if since `$options['sourceCulture']` would just end up being NULL.